tests/ImageProcessorCore.Benchmarks/General/ArrayCopy.cs

```diff
@@ -7,25 +7,44 @@ namespace ImageProcessorCore.Benchmarks.General
 
     public class ArrayCopy
     {
-        private double[] source = new double[10000];
+        [Params(100, 1000, 10000)]
+        public int Count { get; set; }
 
-        [Benchmark(Baseline = true, Description = "Copy using Array.Copy()")]
-        public double CopyArray()
-        {
+        byte[] source, destination;
 
-            double[] destination = new double[10000];
-            Array.Copy(source, destination, 10000);
+        long sizeInBytes;
 
-            return destination[0];
+        [Setup]
```

Comment:
@JimBobSquarePants It was [added recently](https://github.com/PerfDotNet/BenchmarkDotNet/issues/215), it's called `Cleanup`.